queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/alternate_name.py

```diff
@@ -28,6 +28,21 @@
 from entity_filer.utils.legislation_datetime import LegislationDatetime
 
 
+def update_alternate_name_info(
+    alternate_name: AlternateName, legal_entity: LegalEntity, business_info: Dict, filing: Filing
+):
+    """Format and update the alternate_name entity from incorporation filing."""
+    if legal_entity and alternate_name and business_info and filing:
+        alternate_name.name = business_info.get("legalName", None)
```

Comment:
Not sure if we need to handle a case were `legalName` is None. We do [handle this case for legal entities](https://github.com/bcgov/lear/blob/feature-legal-name/queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/legal_entity_info.py#L63) for the incorporation filing process with no NR.